managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -248,6 +248,7 @@ public void operationComplete(ManagedLedgerInfo mlInfo, Stat stat) {
                     if (log.isDebugEnabled()) {
                         log.debug("[{}] Opening legder {}", name, id);
                     }
+                    store.recordRead();
```

Comment:
Actually, right now we are having single historgram to record read and write latency combined. So, do you think we should introduce separate histogram to record read-latency as well?
And for this L251 instance: we are not recording latency because we don't have callback implementation in this scope so, we couldn't record the read-latency here and just record the rate.
```
store.recordRead();
mbean.startDataLedgerOpenOp();
bookKeeper.asyncOpenLedger(id, config.getDigestType(), config.getPassword(), opencb, null);
```